### PR TITLE
Fixed typos across code comments and docs

### PR DESCRIPTION
**Pull request by AndrewDryga**
This is fixed PR based on #1295 without removing trailing spaces.

Commits are split based on path where typo was found. I can reorder them and create two commits:
1. Changes to code comments
2. Changes to docs (only 8 html/xml files)

### CODE REVIEW

File: lib/hipe/opt/hipe_schedule.erl [hunk 1]
```diff
@@ -1344,10 +1344,10 @@ cd([{N,I}|Xs], DAG, PrevBr, PrevUnsafe, PrevOthers) ->
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %% Function    : cd_branch_to_other_deps
 %% Argument    : N   - index of branch
-%%               Ms  - list of indexes of "others" preceeding instrs
+%%               Ms  - list of indexes of "others" proceeding instrs
```
Comment (kostis) [inline lib/hipe/opt/hipe_schedule.erl]: This change is wrong: there is a typo alright, but the right word should be `preceding`.

File: lib/hipe/opt/hipe_schedule.erl [hunk 2]
```diff
@@ -1344,10 +1344,10 @@ cd([{N,I}|Xs], DAG, PrevBr, PrevUnsafe, PrevOthers) ->
...
 %%               DAG - dependence graph
 %% Returns     : DAG - new graph
-%% Description : Makes preceeding instrs fixed so they don't bypass a branch
+%% Description : Makes proceeding instrs fixed so they don't bypass a branch
```
Comment (kostis) [inline lib/hipe/opt/hipe_schedule.erl]: Same comment as above.  It should say `preceding`.

File: erts/etc/unix/etp-commands.in [hunk 3]
```diff
@@ -2596,7 +2596,7 @@ document etp-disasm
 %---------------------------------------------------------------------------
 % etp-disasm StartI EndI
 %
-% Disassemble the code inbetween StartI and EndI
+% Disassemble the code in-between StartI and EndI
```
Comment (bjorng) [inline erts/etc/unix/etp-commands.in]: I think that simply "between" would be more correct in this context.

File: erts/emulator/internal_doc/DelayedDealloc.md [hunk 4]
```diff
@@ -65,29 +65,29 @@ list. In the uncontended case it will point to the end of the list,
...
 pointer. This is done using an atomic compare and swap that expects
-the next pointer to be `NULL`. If this succeds the thread performing
+the next pointer to be `NULL`. If this succeeds the thread performing
 this operation moves the last pointer to point to the newly inserted
 element.
 
 If the atomic compare and swap described above failed, the last
 pointer didn't point to the last element. In this case we need to
-insert the new element somewhere inbetween the element that the last
+insert the new element somewhere in-between the element that the last
```
Comment (bjorng) [inline erts/emulator/internal_doc/DelayedDealloc.md]: Just "between" is more correct here.

File: erts/emulator/sys/unix/erl_child_setup.h [hunk 5]
```diff
@@ -17,7 +17,7 @@
  * 
  * %CopyrightEnd%
  *
- * This file defines the interface inbetween erts and child_setup.
+ * This file defines the interface in-between erts and child_setup.
```
Comment (bjorng) [inline erts/emulator/sys/unix/erl_child_setup.h]: Just "between".

**Comment by bjorng** [comment]
Looks good!

However, we must ask you to rebase this PR on the `master` branch. We only want essential changes in the `maint` branch, to make sure that it is very low risk to use the the changes in the `maint` branch. New features and cosmetic changes should be made in the `master` branch.

Also a minor nitpick about the commit message. Use the imperative style: "Fix typos in..."

**Comment by AndrewDryga** [comment]
@bjorng Done :).

**Comment by IngelaAndin** [comment]
I see that you have put typo fixes for both the ssl and the ssh application under commit Fix 
lib/ssl typos which is not really true as it is typos in lib/ssl and lib/ssh

**Comment by bjorng** [comment]
Thanks!

Please squash the last two correction commits into the original commits. (To simplify that, you could combine the the original four commits involving erts with correction commit into a single commit for erts.)

**Comment by AndrewDryga** [comment]
@IngelaAndin I've spit those commit into two. Thanks.

@bjorng Done 👍.

**Comment by IngelaAndin** [comment]
Looks good from Team-PS perspective now

**Pull request merged**